check_stability.py

```diff
@@ -606,9 +606,8 @@ def main():
     parser = get_parser()
     args = parser.parse_args()
 
-    user_path = os.path.join(os.path.abspath(os.curdir), args.user)
-    wpt_root = os.path.join(user_path, "web-platform-tests")
-    wptrunner_root = os.path.join(user_path, "wptrunner")
+    wpt_root = os.path.join(os.path.abspath(os.curdir))
+    wptrunner_root = os.path.join(wpt_root, "..", "wptrunner")
```

Comment:
Just had that same thought and pushed another change - rebased because I was getting so many fixups.